AutocompleteClient/FW/Logic/Worker/ConstructorIO.swift

```diff
@@ -583,7 +583,9 @@ public class ConstructorIO: CIOSessionManagerDelegate {
 
     private func attachABTestCells(requestBuilder: RequestBuilder) {
         self.config.testCells?.forEach({ [unowned requestBuilder] cell in
-            requestBuilder.set(testCellKey: cell.key, testCellValue: cell.value)
+            if (!cell.key.isEmpty) {
```

Comment:
Should we also check to make sure that the `value` isn't empty either?

Reply:
I thought about it, decided not to since it's somewhat valuable to know if the variant is empty, it doesn't break the request.

Reply:
Ooo, that works for me! 😄